### ISSUE TITLE
 Change the Javascript interpreter engine to Rhino 

### ISSUE THREAD

**Issue by stevefan1999-personal** [MinecraftForge/MinecraftForge#7588]
CC @ https://github.com/Mohist-Community/Mohist/issues/758#issuecomment-754761055

**Comment by stevefan1999-personal** [comment]
@LexManos would you like to give me the reason for closing this, or was it done already?

**Comment by LexManos** [comment]
This is a no-effort spam issue that should of never come here.
I expect a bit of effort and research to be done on your end so you know what you're talking about.
Just posting a link is very disrespectful and shows you care very little for your time or ours.
To answer the question tho, no, and this isn't even the repo for it. And there are many other related issues that have fuller answers and better discussion.

TLDR: Be a better community member and don't spam us with useless things.

**Comment by stevefan1999-personal** [comment]
@LexManos Because I don't want to copy paste the original issue content and the maintainers specifically told me that "Nashorn is used by forge", that's why I ended up here. It's a strategy to save redundant information and I don't think this automatically makes it a spam on sus.

Fine, you just want me to show evidence right? Here is how I prove it:
```pwsh
(base) PS C:\Users\Steve\Desktop\forge> java -version
openjdk version "15.0.1" 2020-10-20
OpenJDK Runtime Environment AdoptOpenJDK (build 15.0.1+9)
Eclipse OpenJ9 VM AdoptOpenJDK (build openj9-0.23.0, JRE 15 Windows 10 amd64-64-Bit Compressed References 20201022_82 (JIT enabled, AOT enabled)
OpenJ9   - 0394ef754
OMR      - 582366ae5
JCL      - ad583de3b5 based on jdk-15.0.1+9)
(base) PS C:\Users\Steve\Desktop\forge> java -jar .\forge-1.16.4-35.1.32.jar // Built using conventional method aka installer.jar --installServer
2021-01-06 18:55:21,059 main WARN Advanced terminal features are not available in this environment
[18:55:21] [main/INFO] [cp.mo.mo.Launcher/MODLAUNCHER]: ModLauncher running: args [--gameDir, ., --launchTarget, fmlserver, --fml.forgeVersion, 35.1.32, --fml.mcpVersion, 20201102.104115, --fml.mcVersion, 1.16.4, --fml.forgeGroup, net.minecraftforge]
[18:55:21] [main/INFO] [cp.mo.mo.Launcher/MODLAUNCHER]: ModLauncher 8.0.6+85+master.325de55 starting: java version 15.0.1 by AdoptOpenJDK
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by cpw.mods.modlauncher.SecureJarHandler (file:/C:/Users/Steve/Desktop/forge/libraries/cpw/mods/modlauncher/8.0.6/modlauncher-8.0.6.jar) to field java.util.jar.Manifest.jv
WARNING: Please consider reporting this to the maintainers of cpw.mods.modlauncher.SecureJarHandler
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[18:55:21] [main/INFO] [ne.mi.fm.lo.FixSSL/CORE]: Added Lets Encrypt root certificates as additional trust
[18:55:21] [main/INFO] [mixin/]: SpongePowered MIXIN Subsystem Version=0.8.2 Source=file:/C:/Users/Steve/Desktop/forge/libraries/org/spongepowered/mixin/0.8.2/mixin-0.8.2.jar Service=ModLauncher Env=SERVER
Exception in thread "main" java.lang.NoClassDefFoundError: jdk.nashorn.api.scripting.NashornScriptEngineFactory
        at net.minecraftforge.coremod.CoreModEngine.loadCoreMod(CoreModEngine.java:48)
        at net.minecraftforge.coremod.CoreModProvider.addCoreMod(CoreModProvider.java:12)
        at net.minecraftforge.fml.loading.LoadingModList$$Lambda$419/0x000000005b1bd520.accept(Unknown Source)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
        at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
        at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:658)
        at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:274)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
        at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:497)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:487)
        at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:239)
        at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
        at net.minecraftforge.fml.loading.LoadingModList.addCoreMods(LoadingModList.java:85)
        at net.minecraftforge.fml.loading.moddiscovery.ModDiscoverer.discoverMods(ModDiscoverer.java:129)
        at net.minecraftforge.fml.loading.FMLLoader.beginModScan(FMLLoader.java:215)
        at net.minecraftforge.fml.loading.FMLServiceProvider.runScan(FMLServiceProvider.java:107)
        at cpw.mods.modlauncher.TransformationServiceDecorator.runScan(TransformationServiceDecorator.java:114)
        at cpw.mods.modlauncher.TransformationServicesHandler.lambda$runScanningTransformationServices$8(TransformationServicesHandler.java:115)
        at cpw.mods.modlauncher.TransformationServicesHandler$$Lambda$268/0x000000005af92a30.apply(Unknown Source)
        at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:271)
        at java.base/java.util.HashMap$ValueSpliterator.forEachRemaining(HashMap.java:1766)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:497)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:487)
        at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:239)
        at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
        at cpw.mods.modlauncher.TransformationServicesHandler.runScanningTransformationServices(TransformationServicesHandler.java:116)
        at cpw.mods.modlauncher.TransformationServicesHandler.initializeTransformationServices(TransformationServicesHandler.java:63)
        at cpw.mods.modlauncher.Launcher.run(Launcher.java:76)
        at cpw.mods.modlauncher.Launcher.main(Launcher.java:66)
        at net.minecraftforge.server.ServerMain$Runner.runLauncher(ServerMain.java:63)
        at net.minecraftforge.server.ServerMain$Runner.access$100(ServerMain.java:60)
        at net.minecraftforge.server.ServerMain.main(ServerMain.java:57)
Caused by: java.lang.ClassNotFoundException: jdk.nashorn.api.scripting.NashornScriptEngineFactory
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:792)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:168)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:1108)
        ... 37 more
(base) PS C:\Users\Steve\Desktop\forge>
```
